src/shared/components/post/metadata-card.tsx

```diff
@@ -75,7 +75,7 @@ export class MetadataCard extends Component<
           </div>
         )}
         {this.state.expanded && post.embed_video_url && (
-          <iframe src={post.embed_video_url}></iframe>
+          <center><iframe allowFullscreen="allowfullscreen" src={post.embed_video_url} style="aspect-ratio:16/9;max-width:85vw" width="560px" frameborder="0"></iframe></center>
```

Comment:
Do not hardcode widths or aspect ratios like this. Use bootstrap classes made for iframes: https://getbootstrap.com/docs/5.3/helpers/ratio/#example

Reply:
FYI the feature you linked appears to be new in Bootstrap v5. As the UI is still using bootstrap v4 the `.ratio` helper is not available yet.

Reply:
@jsit is working on Bootstrap 5: https://github.com/LemmyNet/lemmy-ui/pull/1378

Reply:
I have opened a PR following the recommended approach #1437